kura/org.eclipse.kura.driver.opcua.provider/src/main/java/org/eclipse/kura/internal/driver/opcua/OpcUaDriver.java

```diff
@@ -225,7 +171,13 @@ public synchronized void connect() throws ConnectionException {
                 endPointBuilder.append("/").append(serverName);
             }
 
-            final EndpointDescription[] endpoints = UaTcpStackClient.getEndpoints(endPointBuilder.toString()).get();
+            String endpointString = endPointBuilder.toString();
+
+            logger.debug("Connecting to endpoint: {}", endpointString);
+
+            logger.debug("Fetching endpoint descriptions");
```

Comment:
It's debug, so it's not supposed to be localized. In this way we also prevent the cost of localization for strings that in 95% of cases are not shown.